DATA/SConscript

```diff
@@ -206,205 +370,434 @@ ingest = env.Command(os.path.join(REPO_ROOT, "LATISS", "raw"), butler,
-                        [getPipeTaskCmd('crosstalk', exposureDict['scienceExposures'],
-                                        'cpCrosstalk.yaml'),
-                         getCertifyCmd('crosstalk'),
-                         getVerifyCmd('crosstalk', exposureDict['scienceExposures'],
-                                      'verifyCrosstalk.yaml'),
+if LEGACY_MODE == 0:
+    # New IsrTaskLSST based pipelines.
+
+    # Create bootstrap bias
+    # These are not certified.
+    biasBootstrap = env.Command(
+        [
+            os.path.join(REPO_ROOT, "ci_cpp_biasBootstrap"),
+            os.path.join(REPO_ROOT, "calib", "v00", "biasBootstrap"),
+        ],
+        ingest,
+        [
+            getPipeTaskCmd("biasBootstrap", exposureDict["biasExposures"], "cpBiasBootstrap.yaml"),
+        ]
+    )
+    env.Alias("biasBootstrap", biasBootstrap)
+
+    # Create bootstrap dark
+    # These are not certified.
+    darkBootstrap = env.Command(
+        [
+            os.path.join(REPO_ROOT, "ci_cpp_darkBootstrap"),
+            os.path.join(REPO_ROOT, "calib", "v00", "darkBootstrap"),
+        ],
+        biasBootstrap,
+        [
+            getPipeTaskCmd(
+                "darkBootstrap",
+                exposureDict["darkExposures"],
+                "cpDarkBootstrap.yaml",
+            ),
+        ],
+    )
+    env.Alias("darkBootstrap", darkBootstrap)
+
+    # Create a bootstrap flat.
+    # These are not certified.
+    flatBootstrap = env.Command(
+        [
+            os.path.join(REPO_ROOT, "ci_cpp_flatBootstrap"),
+            os.path.join(REPO_ROOT, "calib", "v00", "flatBootstrap"),
+        ],
+        [biasBootstrap, darkBootstrap],
+        [
+            getPipeTaskCmd(
+                "flatBootstrap",
+                exposureDict["flatExposures"],
+                "cpFlatBootstrap.yaml",
+            ),
+        ],
+    )
+    env.Alias("flatBootstrap", flatBootstrap)
+
+    # Create a defect set and certify.
+    # Defect verification is performed later after other calibs
+    # have been built.
+    defects = env.Command(
+        [
+            os.path.join(REPO_ROOT, "ci_cpp_defects"),
+            os.path.join(REPO_ROOT, "calib", "v00", "defects"),
+        ],
+        [biasBootstrap, darkBootstrap, flatBootstrap],
+        [
+            getPipeTaskCmd("defects", [exposureDict["flatExposures"][0]], "cpDefects.yaml"),
+            getCertifyCmd("defects"),
+        ],
+    )
+    env.Alias("defects", defects)
+
+    # Create a linearizer and certify/verify.
+    linearizer = env.Command(
+        [
+            os.path.join(REPO_ROOT, "ci_cpp_linearizer"),
+            os.path.join(REPO_ROOT, "ci_cpv_linearizer"),
+            os.path.join(REPO_ROOT, "calib", "v00", "linearizer"),
+        ],
+        [defects],
+        [
+            getPipeTaskCmd("linearizer", exposureDict["ptcExposurePairs"], "cpLinearizer.yaml"),
+            getCertifyCmd("linearizer"),
```

Comment:
Should the verify command run before the certify command?  I see that the current code matches what's here, but when we run these for real, the verify is run against the uncertified calibration.  It probably doesn't matter, but if we have connections that can only be met by certified calibrations, then that could cause problems in production.  I'm happy to take any tickets this spawns if something is broken.

Reply:
I've added DM-46432 to look into this, so it doesn't need to complicate the merging here.  That ticket will decide what the right solution is.